src/_scss/pages/agency/index.scss

```diff
@@ -126,11 +126,13 @@
             width: 100%;
             @include flex-direction(column);
             padding: 2.5% 2.5% 0 2.5%;
-            margin: rem(20) rem(20) rem(20) 0;
             background: $color-white;
             @media(min-width: $tablet-screen) {
                 width: 70%;
             }
+            @media(min-width: 992px) {
```

Comment:
should use the $medium-screen variable here instead of 992

Reply:
good call